### PR TITLE
start: Reorganize into Trails (Candidate #1)

### PR DESCRIPTION
**Pull request by iesahin**
This PR splits the GS content into little pages that can be interlinked. It's planned to have 3 introduction points. (Data, Pipelines, Experiments)

### CODE REVIEW

**Comment by iesahin** [comment]
I used #3050 as a base branch for this, but it looks many other commits are here as well. Weird.

File: content/docs/start/pipelines/dag.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Reproduce
+
+The whole point of creating this `dvc.yaml` file is the ability to easily
+reproduce a pipeline:
+
+```dvc
+$ dvc repro
+```
+
+<details>
```
Comment (julieg18) [inline content/docs/start/pipelines/dag.md:85]: Deleting this line should get the branch to deploy :)

```suggestion
```

**Comment by jorgeorpinel** [comment]
Yeah sorry. Can't review this. It's showing 477 files changed and no deployment 😢

**Comment by jorgeorpinel** [comment]
> I used https://github.com/iterative/dvc.org/pull/3050 as a base branch for this

p.s. Let's try to finish that one first or simultaneously please! Same priority 🙂 (high)

**Comment by iesahin** [comment]
Closed in favor of #3444

**Pull request closed without merging**